app/scripts/minmax/dimMinMax.controller.js

```diff
@@ -19,8 +19,16 @@
 
     function getBestArmor(bucket, locked) {
       var armor = {};
+      if (vm.active == 'all'){
```

Comment:
shouldn't need this check since `getBestArmor` won't be called for "all" (per line 360)